### ISSUE TITLE
Server crash on /app/routes/auth.js:18

### ISSUE THREAD

**Issue by adcpm** [adcpm/steemconnect.com#88]
```
2016-11-25T10:36:53.811711+00:00 app[web.1]: ::ffff:10.184.171.93 - - [25/Nov/2016:10:36:53 +0000] "GET /api/@busy.app/verify HTTP/1.1" 401 12 "https://busy6-pr-148.herokuapp.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2926.0 Safari/537.36"
2016-11-25T10:37:22.568146+00:00 heroku[router]: at=error code=H13 desc="Connection closed without response" method=GET path="/logout" host=dev.steemconnect.com request_id=d28f7041-932c-459f-bcf4-c946cdff9b6f fwd="87.88.231.248,162.158.22.63" dyno=web.1 connect=1ms service=26ms status=503 bytes=0
2016-11-25T10:37:22.555416+00:00 app[web.1]: /app/routes/auth.js:18
2016-11-25T10:37:22.555425+00:00 app[web.1]:       const username = jwtData.username;
2016-11-25T10:37:22.555426+00:00 app[web.1]:                               ^
2016-11-25T10:37:22.555427+00:00 app[web.1]: 
2016-11-25T10:37:22.555428+00:00 app[web.1]: TypeError: Cannot read property 'username' of undefined
2016-11-25T10:37:22.555428+00:00 app[web.1]:     at jwt.verify.e (/app/routes/auth.js:18:31)
2016-11-25T10:37:22.555429+00:00 app[web.1]:     at /app/node_modules/jsonwebtoken/verify.js:27:18
2016-11-25T10:37:22.555430+00:00 app[web.1]:     at _combinedTickCallback (internal/process/next_tick.js:67:7)
2016-11-25T10:37:22.555431+00:00 app[web.1]:     at process._tickCallback (internal/process/next_tick.js:98:9)
2016-11-25T10:37:22.578908+00:00 app[web.1]: 
2016-11-25T10:37:22.596472+00:00 app[web.1]: npm ERR! Linux 3.13.0-100-generic
2016-11-25T10:37:22.596726+00:00 app[web.1]: npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "start"
2016-11-25T10:37:22.596897+00:00 app[web.1]: npm ERR! node v6.9.1
2016-11-25T10:37:22.597064+00:00 app[web.1]: npm ERR! npm  v3.10.8
```

**Comment by nil1511** [comment]
How to reproduce this bug?

**Comment by adcpm** [comment]
Not sure. I saw server crash after login and before logout.

**Comment by nil1511** [comment]
Closed in 5b304ef
